react_main/src/pages/Game/Game.jsx

```diff
@@ -1613,6 +1613,7 @@ export function ActionList(props) {
                 case "boolean":
                 case "role":
                 case "alignment":
+                case "custom":

```

Comment:
It forces the inputType to use ActionSelect (i.e. a dropdown menu).

It would work if I used "alignment" here instead of "custom", but that it is semantically incorrect